Fabric.Authorization.IntegrationTests/Modules/UserTests.cs

```diff
@@ -1356,5 +1356,19 @@ public void TestUserWhitelist_Success()
             Assert.Contains("app/userprincipal.modifypatient", permissions.Permissions);
             Assert.Equal(3, permissions.Permissions.Count());
         }
+
+        [Fact]
+        [DisplayTestMethodName]
+        public void Test_GetPermissions_UserNotFound()
```

Comment:
Should this test be named "GetGroups" instead of "GetPermissions"?

Reply:
Yes it should, good catch.